.travis.yml

```diff
@@ -42,8 +42,6 @@ matrix:
     - os: linux
       env: PROJECT_DIR=examples/OpenCL TOOLCHAIN=gcc-4-8
     - os: linux
-      env: PROJECT_DIR=examples/OpenCL TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
```

Comment:
Not sure what you mean? Do you want me to comment out the tool chain and add a remark as to why it is not run?